packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -197,7 +197,7 @@
 	border-radius: $radius-round;
 	box-shadow: inset 0 0 0 $border-width rgba(0, 0, 0, 0.2);
 	// Show a thin outline in Windows high contrast mode, otherwise the button is invisible.
-	border: 1px solid transparent;
```

Comment:
This one is fine to change, but there are many cases where there are 2 or even 3px outlines destined for high contrast mode, so it's less uniform here. In other words, this one could intentionally diverge. Either is fine, just noting.